crypto/dh_extra/dh_test.cc

```diff
@@ -442,11 +566,13 @@ TEST(DHTest, RFC7919) {
 
 TEST(DHExpectedTestnputTest, CalculateSharedSecretMatches) {
   // KAT calculated with the following sage math code:
-  // prime=int("0x[prime for 2048 or 4096]", 16) R=Integers(prime) g = R(2)
-  // client_sk = int("0xABCDEF1234567890ABCDEF1234567890ABCDEF1234567890ABCDEF1234567890", 16) client_pk = g^client_sk
-  // server_sk = int("0xAABBCCDDEEFF11223344556677889900AABBCCDDEEFF11223344556677889900", 16) shared_secret = client_pk^server_sk
-  // print("client_pk", format(int(client_pk), '#x'))
-  // print("server_sk", format(server_sk, '#x'))
+  // prime=int("0x[prime for 2048 - 8192]", 16) R=Integers(prime) g = R(2)
+  // client_sk =
+  // int("0xABCDEF1234567890ABCDEF1234567890ABCDEF1234567890ABCDEF1234567890",
+  // 16) client_pk = g^client_sk server_sk =
+  // int("0xAABBCCDDEEFF11223344556677889900AABBCCDDEEFF11223344556677889900",
```

Comment:
looks like clang-format messed up some whitespace-sensitive syntax?

Reply:
Ah yes, thanks for noticing!